addons/website_slides/static/src/xml/website_slides_upload.xml

```diff
@@ -188,4 +291,14 @@
             </form>
         </div>
     </t>
+
+    <!-- Misc -->
+    <t t-name="website.slide.upload.modal.loading">
+        <div class="o_wslides_slide_upload_loading position-absolute h-100 w-100 text-center d-flex flex-column justify-content-center">
+            <h2 class="text-white">
+                <span class="fa fa-spinner fa-pulse"></span>
+                <span>Loading slide information...</span>
```

Comment:
slide ? not "lesson" ? or juste "Loading information" maybe ?

Reply:
Ah ! Luc was on the hunt for those, we replaced a bit everywhere for "content", good catch !